src/components/LocationSelector.vue

```diff
@@ -127,8 +128,29 @@ export default {
     toggleOptions() {
       this.optionsShowed = !this.optionsShowed;
     },
+    updateCountryCodeParam(country) {
+      this.$router.push({
```

Comment:
```suggestion
      this.$router.push({```suggestion
      const query = country && country.code !== 'global' ? {
        country_code: country.code,
      } : null;
      
      this.$router.push({
         path: '/',
        query,
      });
```
```